studio/structure/langauages.ts

```diff
@@ -0,0 +1,5 @@
+//TODO: create a schema type for these
+export const supportedLanguages = [
+  {id: 'en-US', title: 'English (US)'},
+  {id: 'mi-NZ', title: 'Māori'}
+]
```

Comment:
:memo: this will become a `supportedLanguages` schema, in which we can dynamically add and query from

Reply:
🎉 Nice, thanks for adding this ticket!